fiware-region-sanity-tests/tests/fiware_region_without_networks_tests.py

```diff
@@ -184,3 +187,22 @@ def test_deploy_instance_and_e2e_connection_using_public_ip(self):
 
         ## SSH Connection
         self.__ssh_connection_test_helper__(host=allocated_ip, private_key=private_keypair_value)
+
+    def test_deploy_instance_and_e2e_snat_connection(self):
+        """
+        Test whether it is possible to deploy an instance and connect to INTERNET (PhoneHome service)
+        """
+
+        # Deploy
+        suffix = datetime.utcnow().strftime('%Y%m%d%H%M%S')
+        instance_name = TEST_SERVER_PREFIX + "_snat_" + suffix
+        server_id = self.__deploy_instance_helper__(instance_name=instance_name,
+                                                    userdata=PHONEHOME_USERDATA_PATH)
+
+        # Create and launch the PhoneHome service. Wait for request from VM
+        http_phonehome_server = HttpPhoneHomeServer(logger=self.logger, port=PHONEHOME_PORT, timeout=PHONEHOME_TIMEOUT)
+        http_phonehome_server.start()
+
+        self.assertIsNotNone(get_phonehome_content(), "Phone-Home request not received from VM '%s'" % server_id)
```

Comment:
I think that the test is well done; only as suggestion, check that the received content is the sent one, instead of only testing that is not empty. I'll include also a secret token in the string.

As I said, these are only suggestions. I think that the current code do its work.